pkg/apis/core/types.go

```diff
@@ -2099,10 +2099,14 @@ type NodeSelector struct {
 	NodeSelectorTerms []NodeSelectorTerm
 }
 
-// A null or empty node selector term matches no objects.
+// A null or empty node selector term matches no objects. At least one of
```

Comment:
remove `At least one of 'MatchExpressions' and 'MatchFields' must be specified`, since the conflicts with `A null or empty node selector term matches no objects.`, and we want a 1.10 server to tolerate a `NodeSelectorTerm` containing no MatchExpressions field.